packages/react-native/Libraries/LogBox/Data/LogBoxData.js

```diff
@@ -30,6 +30,7 @@ export type LogData = $ReadOnly<{|
   message: Message,
   category: Category,
   componentStack: ComponentStack,
+  codeFrame?: string,
```

Comment:
Tried to add the codeFrame to the `addLog` call, but it is clearly not working. Any ideas how to replace the source frame?